Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 75, in execute
    loop.run_until_complete(future)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 603, in run_forever
    self._run_once()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 1871, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.10/selectors.py", line 469, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__
    rc = await self.task(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__
    return await task_method(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_cmake/build.py", line 65, in build
    has_install_target = await has_target(args.build_base, 'install')
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/__init__.py", line 57, in has_target
    return target in await get_makefile_targets(path)
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/__init__.py", line 77, in get_makefile_targets
    output = await check_output([
  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 129, in check_output
    assert not rc, f'Expected {args} to pass: {stderr_data}'
AssertionError: Expected ['/home/robot/.local/bin/cmake', '--build', '/home/robot/ws_moveit2/build/moveit_common', '--target', 'help'] to pass: Traceback (most recent call last):
  File "/home/robot/.local/bin/cmake", line 5, in <module>
    from cmake import cmake
  File "/home/robot/.local/lib/python3.10/site-packages/cmake/__init__.py", line 29, in <module>
    cmake_files = distribution("cmake").files
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 653, in files
    return file_lines and list(starmap(make_file, csv.reader(file_lines)))
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 646, in make_file
    def make_file(name, hash=None, size_str=None):
KeyboardInterrupt

